Files/ViewModels/CurrentInstanceViewModel.cs

```diff
@@ -71,28 +76,35 @@ public bool IsPageTypeRecycleBin
                 SetProperty(ref _IsPageTypeRecycleBin, value);
                 OnPropertyChanged(nameof(IsCreateButtonEnabledInPage));
                 OnPropertyChanged(nameof(CanCreateFileInPage));
+                OnPropertyChanged(nameof(CanPasteInPage));
                 OnPropertyChanged(nameof(CanOpenTerminalInPage));
+                OnPropertyChanged(nameof(CanCopyPathInPage));
             }
         }
 
         public bool IsCreateButtonEnabledInPage
         {
-            get => !_IsPageTypeRecycleBin && IsPageTypeNotHome && !_IsPageTypeSearchResults;
+            get => !_IsPageTypeRecycleBin && _IsPageTypeNotHome && !_IsPageTypeSearchResults;
```

Comment:
```suggestion
            get => !isPageTypeRecycleBin && isPageTypeNotHome && !isPageTypeSearchResults;
```

Reply:
I know they have all the wrong names (this code is old) wanted to keep unnecessary changes to a minimum.
This is pretty small though, I'll do it 👍

Reply:
@gave92 Thanks!